docs/README.md

```diff
@@ -48,7 +52,7 @@ p4c
 * the P4_14 (P4 v1.0.4) language is described in the P4 spec:
   https://p4lang.github.io/p4-spec/p4-14/v1.0.4/tex/p4.pdf
 
-* the P4_16 language (v1.0.0) is described in [this pdf
+* the P4_16 language (v1.1.0) is described in [this pdf
   document](https://p4lang.github.io/p4-spec/docs/P4-16-v1.0.0-spec.pdf)
```

Comment:
Please update the URL in the link as well:
https://p4.org/p4-spec/docs/P4-16-v1.1.0-draft.pdf